ingest.py

```diff
@@ -0,0 +1,310 @@
+import sys
+import os
+import shutil
+import re
+from datetime import datetime, timezone
+from PIL import Image, ImageCms
+import io
+import logging
+import subprocess
+import json
+
+from variables import SRC, DST, SKIPPED, valid_id_initial_chars, valid_suffixes, valid_first_segment_first_char, valid_first_segment_other_chars, required_metadata_tags
+
+IMAGE_EXTENSIONS = ('.jpg', '.jpeg', '.png', '.gif', '.bmp', '.tif', '.tiff')
+
+now = datetime.now(timezone.utc).astimezone()
+date_suffix = now.strftime("%Y-%m-%dT%H%M%S")
+date_isoformat = now.replace(microsecond=0).isoformat()
+
+log_directory = os.path.join(DST, "__log__")
+os.makedirs(log_directory, exist_ok=True)
+log_file_name = f"script_{date_suffix}.log"
+log_path = os.path.join(log_directory, log_file_name)
+logging.basicConfig(filename=log_path, level=logging.INFO, format='%(asctime)s [%(levelname)s] %(message)s')
+
+logging.info("Start ingest process")
+logging.info(f"Source directory = {SRC}")
+logging.info(f"Destination directory = {DST}")
+
+def move_file(src, dst, reason):
+    try:
+        shutil.move(src, dst)
+        logging.info(f"Moved: {src} -> {dst} ({reason})")
+    except shutil.Error as e:
+        logging.error(f"Error moving {src} to {dst}: {e}")
+
+def is_image_file(file_name):
```

Comment:
I would introduce doctests for all these functions here to ensure they do what they are intended to do even in corner cases.
```suggestion
def is_image_file(file_name: str):
"""
  Is valid image file name
  
  >>> is_image_file("a.jpg")
  True
  >>> is_image_file("a.txt")
  False
"""
```
Additionally type annotations are very useful for readability.